docs/usage/usage.md

```diff
@@ -74,6 +74,7 @@ Other than tools, preprocessing steps usually don’t return an easily interpret
     :toctree: preprocessing
     :nosignatures:
 
+    preprocessing.sample
```

Comment:
There is now `preprocessing.sample` and `preprocessing.subsample`. https://ehrapy.readthedocs.io/en/latest/usage/preprocessing/ehrapy.preprocessing.subsample.html#ehrapy.preprocessing.subsample is this confusing?

Reply:
Yeah, fair argument.

Reply:
I agree that having both `ep.pp.sample` and `ep.pp.subsample` is confusing. I think calling it `balancing` is better, but I'm still not 100% convinced. Combining both, i.e., `balanced_sampling`, is too long?

Reply:
We should harmonize the names -> we have `subsample` so maybe `balancedsample` or `balanced_sample`?

Reply:
nice inputs, thanks -> both + harmonizing should do